featurex/converters/api.py

```diff
@@ -1,12 +1,21 @@
 import os
-from featurex.stimuli.text import ComplexTextStim
+import base64
+import json
+from featurex.stimuli.text import DynamicTextStim, ComplexTextStim
 from featurex.converters.audio import AudioToTextConverter
 
 try:
     import speech_recognition as sr
 except ImportError:
     pass
 
+try:
```

Comment:
I think you can replace the urllib 2 vs. 3 imports with the version in `six.moves.urllib`, which wraps both versions in Python 3's API. We already have six as a dependency anyway, so I don't see a downside.